modules/src/relayer/ics18_relayer/context.rs

```diff
@@ -1,7 +1,7 @@
 use crate::prelude::*;
 use ibc_proto::google::protobuf::Any;
```

Comment:
I think we can delete this `ics18_relayer` folder, it's meant for hermes.

Reply:
agreed